From 9f24691c39f982ce23bb3e6bfaacd025ceecf7c7 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Thu, 6 Sep 2018 12:46:37 +0530
Subject: [PATCH 502/909] arm64: dts: r8a77990-ebisu: enable USB3.0 peripheral

commit b36c0cd2688d691f2c7cdb3b403fa690361026f3 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch enables USB3.0 peripheral device for the Ebisu board on the
R8A77990 SoC.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
index 0630c3e32d20..9ae0467a3246 100644
--- a/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77990-ebisu.dts
@@ -539,6 +539,10 @@
 	status = "okay";
 };
 
+&usb3_peri0 {
+	status = "okay";
+};
+
 &vin4 {
 	status = "okay";
 };
-- 
2.17.1

